lang/julia/Portfile

```diff
@@ -26,9 +26,9 @@ github.tarball_from releases
 distfiles           ${name}-${version}-full${extract.suffix}
 
 checksums           ${name}-${version}-full${extract.suffix} \
-                    rmd160  5dacf46cf98aea29ec6ef11ce461ced4ee6ab8eb \
-                    sha256  dbfb8cd544b223eff70f538da7bb9d5b6f76fd0b00dd2385e6254e74ad4e892f \
-                    size    138223428
+                    rmd160  87c637fa1125710deb65d74ad9dc04a0fcd141a9 \
+                    sha256  1b05f42c9368bc2349c47363b7ddc175a2da3cd162d52b6e24c4f5d4d6e1232c \
+                    size    153634233
```

Comment:
Oops, the checksum didn't change and is breaking your build.

Reply:
that's because @queensferryme didn't change the version number to `1.6.0` in the Portfile on line 11 - that's certainly needed for upgrading ;)

Reply:
Thx, that was a mistake.